packages/api-client/src/Certificate/Certificate.spec.ts

```
@@ -20,9 +19,12 @@ describe('Certificate', () => {
     it('sends data as FormData', async () => {
       await Certificate.create({ filepath });
 
-      expect(requestMock.mock.calls[0][0].headers['content-type']).toContain(
-        'multipart/form-data'
-      );
+      expect(requestMock.mock.calls[0][0].data).toBeInstanceOf(FormData);
+      const { headers } = requestMock.mock.calls[0][0];
+      const contentType =
+        headers['content-type'] ?? headers['Content-Type'] ?? headers.get?.('Content-Type');
```

ESLint

[error] Replace `·headers['Content-Type']·??` with `⏎········headers['Content-Type']·??⏎·······`

(prettier/prettier)